frontend/src/components/MeetingRoom/ToolbarOverflowButton/ToolbarOverflowButton.spec.tsx

```diff
@@ -55,8 +60,9 @@ describe('ToolbarOverflowButton', () => {
   });
 
   it('should have the unread messages badge present', () => {
-    render(<ToolbarOverflowButton />);
+    render(<ToolbarOverflowButton toggleShareScreen={vi.fn()} isSharingScreen={false} />);
 
-    expect(screen.getByTestId('chat-button-unread-count')).toBeInTheDocument();
+    // We expect the ChatButton in the ToolbarOverflowMenu and the ToolbarOverflowButton to have an unread messages badge present
+    expect(screen.queryAllByTestId('chat-button-unread-count').length).toBe(2);
```

Comment:
@cpettet @behei-vonage 
Maybe out of scope for this PR but I think we should be hiding the badge on the menu button when the menu is open and we already see the chat badge

Reply:
OOS for this one, but the ticket number is VIDCS-3490